components/more-less/more-less.js

```diff
@@ -91,6 +91,48 @@ export class D2LMoreLess extends LocalizeMixin(LitElement)  {
 		`;
 	}
 
+	getLanguage(lang, backupLang) {
+		return this._tryGetResources(lang)
+			|| this._tryGetResources(backupLang)
+			|| this._tryGetResources('en-us');
+	}
+
+	async getLangResources(lang) {
+		var proto = this.constructor.prototype;
+		this.checkLocalizationCache(proto);
+
+		var namespace = `more-less:${lang}`;
+
+		if (proto.__localizationCache.requests[namespace]) {
+			return;
+		}
+
+		proto.__localizationCache.requests[namespace] = true;
+		return this.__langResources[lang];
+	}
+
+	_tryGetResources(val) {
```

Comment:
It's unfortunate that each of the components would need to re-implement this logic. I wonder if maybe the mixin could do some of this before calling `getLanguage` and pass it a bit more information? Like it could null-check and lowercase everything and split the language codes. Then maybe pass in-order array of codes to use? Like `lang` then `langBase` then `backupLang` then `backupLangBase`? Or something.